### ISSUE TITLE
Broken `includeLaunchClasspath=false` on Windows

### ISSUE THREAD

**Issue by szpak** [hcoles/pitest#179]
Issue originally reported to [gradle-pitest-plugin (#26)](https://github.com/szpak/gradle-pitest-plugin/issues/26). After code analyze it turned out to be a problem in PIT with parsing and comparing launch classpath.

The problem is that with `includeLaunchClasspath=false` classpath elements are taken with `ClassPath.getClassPathElements()` which returns elements as were provided in `-cp` (e.g. `c:\path\pitest-1.1.5.jar`). It is later compared in `PluginFilter` with `getClass().getProtectionDomain().getCodeSource().getLocation().getFile()` which returns path from URI (e.g. `/C:/path/pitest-1.1.5.jar`). A comparison of strings fails and `pitest.jar` is not on a classpath which ends on Windows with:

```
Exception in thread "main" org.pitest.util.PitError:
Unable to load class content for org.pitest.boot.HotSwapAgent
```

In addition there could be problem also on Linux where symlinks are used or launch classpath elements are passed as relative paths. One of the solutions is to compare paths as canonical paths.

**Comment by hcoles** [comment]
Thanks @szpak I'll take a look at this as soon as I get chance.

**Comment by szpak** [comment]
Closing as a fix is available in PIT 1.1.5.
